docs/modules/ROOT/pages/api/introspection.adoc

```diff
@@ -50,10 +50,10 @@ on how to compute this ID.
 use openzeppelin::introspection::src5::SRC5;
 ```
 
-SRC5 contract implementation extending xref:ISRC5[`ISRC5`].
+SRC5 component extending xref:ISRC5[`ISRC5`].
 
 [.contract-index]
-.External Functions
+.Embeddable Implementations
```

Comment:
In the Ownable API, the index name is `Embeddable Implementations Functions`. We should normalize

Reply:
Let's stick with just Embeddable Implementations then if you agree.